### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by liviarett**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (alanclarke) [inline .babelrc:1]: plz remove this, we don't support babel and shouldn't be using it here

File: components/Rating.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    offset={`${filledStars}%`}
+                  />
+                  <stop
+                    className='qb-rating-stars-bg'
+                    offset={`${emptyStars}%`}
+                  />
+                </linearGradient>
+                <path
+                  fill='url(#grad-11518185-90.0)'
+                  d='M8 12.5 3.004 16 4.403 10 0 6 5.777 5.44 8 0 10.223 5.44 16 6 11.597 10 12.996 16 M25 12.5 20.004 16 21.403 10 17 6 22.777 5.44 25 0 27.223 5.44 33 6 28.597 10 29.996 16 M42 12.5 37.004 16 38.403 10 34 6 39.777 5.44 42 0 44.223 5.44 50 6 45.597 10 46.996 16 M59 12.5 54.004 16 55.403 10 51 6 56.777 5.44 59 0 61.223 5.44 67 6 62.597 10 63.996 16 M76 12.5 71.004 16 72.403 10 68 6 73.777 5.44 76 0 78.223 5.44 84 6 79.597 10 80.996 16'
```
Comment (alanclarke) [inline components/Rating.js:43]: i wonder if there is a more succinct way to express a star in svg 🤔

File: package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "preact": "^10.5.13"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-function-bind": "^7.14.5",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/register": "^7.14.5",
```
Comment (alanclarke) [inline package.json:25]: remove babel plz

**Review by alanclarke — APPROVED** [review]

File: components/Rating.js (hunk 2 above)
Comment (liviarett) [reply components/Rating.js:43]: probs

**Pull request merged**